rtv/subscriptions.py

```diff
@@ -0,0 +1,72 @@
+import curses
+import sys
+import time
+import logging
+
+from .content import SubscriptionContent
+from .page import BasePage, Navigator, BaseController
+from .curses_helpers import (Color, LoadScreen, add_line)
+
+__all__ = ['SubscriptionController', 'SubscriptionPage']
+_logger = logging.getLogger(__name__)
+
+class SubscriptionController(BaseController):
+    character_map = {}
+
+class SubscriptionPage(BasePage):
+    def __init__(self, stdscr, reddit):
+        self.controller = SubscriptionController(self)
+        self.loader = LoadScreen(stdscr)
+
+        content = SubscriptionContent.get_list(reddit, self.loader)
+        super(SubscriptionPage, self).__init__(stdscr, reddit, content)
+
+    def loop(self):
+        "Main control loop"
+
+        self.active = True
+        while self.active:
+            self.draw()
+            cmd = self.stdscr.getch()
+            self.controller.trigger(cmd)
+
+    @SubscriptionController.register(curses.KEY_F5, 'r')
+    def refresh_content(self):
+        "Re-download all subscriptions and reset the page index"
+
+        self.content = SubscriptionContent.get_list(self.reddit, self.loader)
+        self.nav = Navigator(self.content.get)
+
+    @SubscriptionController.register(curses.KEY_ENTER, 10, curses.KEY_RIGHT)
+    def open_selected_subreddit(self):
+        "Open the selected subreddit"
+
+        from .subreddit import SubredditPage
+        data = self.content.get(self.nav.absolute_index)
+        page = SubredditPage(self.stdscr, self.reddit, data['name'][2:]) # Strip the leading /r
```

Comment:
You shouldn't need to strip the `/r/`, SubredditContent.from_name() does that for you!